inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/load/StarRocksLoadNode.java

```diff
@@ -171,4 +175,20 @@ public String genTableName() {
     public String getPrimaryKey() {
         return primaryKey;
     }
+
+    @Override
+    public List<FieldInfo> addMetaFields(List<FieldInfo> fieldInfos) {
+        List<String> fieldNames = fieldInfos.stream().map(FieldInfo::getName).collect(Collectors.toList());
+        if (!fieldNames.contains(MetaField.INLONG_DATA_TIME.name())) {
+            fieldInfos.add(0, new FieldInfo("inlong_data_time", new LongFormatInfo()));
+        }
+        return fieldInfos;
+    }
+
+    @Override
+    public List<FieldInfo> metaFields() {
+        List<FieldInfo> fieldInfos = new ArrayList<>();
+        fieldInfos.add(0, new FieldInfo("inlong_data_time", new LongFormatInfo()));
```

Comment:
There is one constant named "inlong_data_time"

Reply:
Done